split-the-stream/README.md

```diff
@@ -91,6 +91,39 @@ We have multiple options:
 - To share the code between the upstream and source project, we need to create some shared libraries.
   - Can lead to 3. and/or having another project on our dependency chain.
 
+### Database
+
+We have multiple ways how we can manage database schema after the split:
+
+#### One schema for all
+
+- One schema that works for all use-cases.
+- The schema is defined in one place.
+- Databases can contain unused tables.
+- Schema can be more complicated.
+
+#### Independent schemas
+
+- Each schema fits the use-case.
+- Common changes are harder to share.
+
+#### Multiple alembic branches
```

Comment:
it seems this would be a nightmare to maintain - if we only have 2 repos, 'one schema for all' seems like the easiest to maintain